app/src/main/java/org/simple/clinic/storage/Migration_35_36.kt

```diff
@@ -0,0 +1,204 @@
+        "deletedAt" TEXT,
+        PRIMARY KEY("uuid")
+      )
+    """)
+
+    database.execSQL("""
+      INSERT INTO "Appointment"(
+        "uuid", "patientUuid", "facilityUuid",
+        "scheduledDate", "status", "cancelReason",
+        "remindOn",  "agreedToVisit", "appointmentType",
+        "syncStatus", "createdAt",  "updatedAt",  "deletedAt"
+      )
+      SELECT
+        "uuid", "patientUuid", "facilityUuid",
+        "scheduledDate", "status", "cancelReason",
+        "remindOn",  "agreedToVisit", "appointmentType",
+        "syncStatus", "createdAt",  "updatedAt",  "deletedAt"
+      FROM "Appointment_v35"
+      """
+    )
+    database.execSQL("""DROP TABLE "Appointment_v35" """)
+  }
+
+  private fun migrateMedicalHistories(database: SupportSQLiteDatabase) {
+    database.execSQL("""ALTER TABLE "MedicalHistory" RENAME TO "MedicalHistory_V35" """)
+    database.execSQL("""
+      CREATE TABLE IF NOT EXISTS "MedicalHistory"
+      ("uuid" TEXT NOT NULL,
+      "patientUuid" TEXT NOT NULL,
+      "diagnosedWithHypertension" TEXT NOT NULL,
+      "isOnTreatmentForHypertension" TEXT NOT NULL,
+      "hasHadHeartAttack" TEXT NOT NULL,
+      "hasHadStroke" TEXT NOT NULL,
+      "hasHadKidneyDisease" TEXT NOT NULL,
+      "hasDiabetes" TEXT NOT NULL,
+      "syncStatus" TEXT NOT NULL,
+      "createdAt" TEXT NOT NULL,
+      "updatedAt" TEXT NOT NULL,
+      "deletedAt" TEXT,
+      PRIMARY KEY("uuid"))
+    """)
+
+    database.execSQL("""
+    INSERT OR REPLACE INTO "MedicalHistory"("uuid","patientUuid","diagnosedWithHypertension","isOnTreatmentForHypertension","hasHadHeartAttack","hasHadStroke","hasHadKidneyDisease","hasDiabetes","syncStatus","createdAt","updatedAt", "deletedAt")
+    SELECT "uuid","patientUuid","diagnosedWithHypertension","isOnTreatmentForHypertension","hasHadHeartAttack","hasHadStroke","hasHadKidneyDisease","hasDiabetes","syncStatus","createdAt","updatedAt", "deletedAt"
+    FROM "MedicalHistory_V35"
+    """)
+
+    database.execSQL("""DROP TABLE "MedicalHistory_V35" """)
+  }
+
+  private fun migratePatientAddress(database: SupportSQLiteDatabase) {
+    database.execSQL("""ALTER TABLE "PatientAddress" RENAME TO "PatientAddress_V35" """)
+    database.execSQL("""
+      CREATE TABLE IF NOT EXISTS "PatientAddress" (
+      "uuid" TEXT NOT NULL,
+      "colonyOrVillage" TEXT,
+      "district" TEXT NOT NULL,
+      "state" TEXT NOT NULL,
+      "country" TEXT,
+      "createdAt" TEXT NOT NULL,
+      "updatedAt" TEXT NOT NULL,
+      "deletedAt" TEXT,
+      PRIMARY KEY("uuid")
+      )
+    """)
+
+    database.execSQL("""
+      INSERT OR REPLACE INTO "PatientAddress"("uuid", "colonyOrVillage","district", "state", "country", "createdAt", "updatedAt", "deletedAt")
+      SELECT "uuid", "colonyOrVillage","district", "state", "country", "createdAt", "updatedAt", "deletedAt"
+      FROM "PatientAddress_V35"
+    """)
+
+    database.execSQL("""DROP TABLE "PatientAddress_V35" """)
```

Comment:
This is weird. This should've ended up deleting the Patient table but it didn't.

Reply:
Do you think it is because of foreign key constraints being disabled by default?

Reply:
I don't think that's the case because they've always been enabled by default.

I ran your branch by generating data on `master` then applying your PR on top. It seems good, so I am going to merge it in. We can investigate this later.

Reply:
Yea the migration works properly. I also updated the app and tried.